src/components/Counter.component.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+// import PropTypes from 'prop-types'
+import './../styles/instructions.css'
+
+class Counter extends Component {
+  state = { count: this.props.initialCount }
```

Comment:
for `App.js`, you set state inside the constructor, but here you did this instead.
Either way is fine, but just be consistent